documentation/docker.asciidoc

```diff
@@ -28,6 +28,8 @@ The arguments (`devon docker «args»`) are explained by the following table:
 |=======================
 
 == setup
+
+=== Choosing either Docker Desktop or Rancher Desktop for installation
```

Comment:
we have way too many sections in this guide. Do we need this heading?
IMHO this would be sufficient:
1. Setup
...explain that you can choose between DockerDesktop and RancherDesktop...
2. RancherDesktop
...explain details...
3. DockerDesktop
...

Reply:
I removed some subsections now so that we have a shorter structure.